data/json/items/melee.json

```diff
@@ -2356,5 +2356,23 @@
     "price": 4000,
     "price_postapoc": 8000,
     "qualities": [ [ "HAMMER", 1 ] ]
+  },
+  {
+    "id": "lucern_hammer",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "lucern hammer",
```

Comment:
Inferior and fake versions going to be added with this hammer?

Reply:
it's kind of an obscure weapon, so I don't know if anybody would botter making cheap versions.

Reply:
The question is more "would the technique work with that weapon irl?" rather then "will people use it?" I think.

Reply:
What?

Reply:
I though there were already some cheap version of the hammer and you were debating on adding them to the technique. But in fact those cheaper version don't exist, sorry.